src/docs.js

```diff
@@ -138,6 +138,11 @@ async function documenter(options) {
       });
     });
 
+  function getDocumentationUrl() {
+    let referenceUrl = "https://docs.taskcluster.net/reference/";
```

Comment:
Can we set `referenceUrl` as an option with a default [here](https://github.com/taskcluster/taskcluster-lib-docs/blob/master/src/docs.js#L15)? That way we can override this if we want to at a later point.

I believe the reason the tests are currently failing is due to the use of double quotes here! Switching to single quotes should clear it up :)